content/applications/finance/accounting/taxes/avatax.rst

```diff
@@ -101,8 +101,8 @@ If this is an additional license key, ensure the previous connection can be brok
 **only** one license key associated with each of the Avalara sandbox and production accounts.
 
 .. important::
-   Copy this key to a safe place. It is **strongly encouraged** to backup the license key for future
-   reference. This key cannot be retrieved after leaving this screen.
+   Copy this key to a safe place. It is **strongly encouraged** to back up the license key for
```

Comment:
I was also going to suggest that the note above (.. note::
   A warning appears stating: `If your business app is connected to Avalara solutions, the
   connection will be broken until you update the app with the new license key. This action cannot
   be undone.`) be changed to an Important, since if they don't follow that rule, everything breaks- that's what it sounds like to me. So I feel that is important. 

This specific important block is SO important, I feel like this could be a warning. Since if it's gone, it's GONE. Forever! But these are just suggestions and not necessary,